tests/stl_import_test.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""Trimesh and Radia stl import test
+
+:copyright: Copyright (c) 2017-2022 RadiaSoft LLC. All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+
+_RAW_VERTICES = [[0, 0, 0], [1, 1, 0], [0, 1, 0], [0.5, 0.5, 1]]
+_RAW_FACES = [[1, 2, 3], [1, 2, 4], [1, 4, 3], [4, 2, 3]]
+
+RAW_EXPECTED = [[0.0, 1.0, 0.0],
+ [1.0, 1.0, 0.0],
+ [0.0, 0.0, 0.0],
+ [0.0, 0.0, 0.0],
+ [1.0, 1.0, 0.0],
+ [0.5, 0.5, 1.0],
+ [0.0, 0.0, 0.0],
+ [0.5, 0.5, 1.0],
+ [0.0, 1.0, 0.0],
+ [0.5, 0.5, 1.0],
+ [1.0, 1.0, 0.0],
+ [0.0, 1.0, 0.0]]
+
+def _create_mesh(file_path):
+    import trimesh
+    
+    with open(file_path) as f:
+        return trimesh.load(f, file_type='stl', force='mesh', process=True)
```

Comment:
I'm not sure why the formatter does not catch these, but for in any case use double quotes `file_type="stl"...`